src/d3d12/d3d12_renderer.hpp

```diff
@@ -96,10 +101,14 @@ namespace wr
 
 		// temporary
 		d3d12::Heap<HeapOptimization::SMALL_BUFFERS>* m_cb_heap;
+		d3d12::Heap<HeapOptimization::BIG_STATIC_BUFFERS>* m_sb_heap;
```

Comment:
You should now have access to koen's platform independent pool's so you can put this in the scene graph instead.